pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -199,11 +199,13 @@ def _set_dimensional_parameters(self):
 
     def D_e_dimensional(self, c_e, T):
         """Dimensional diffusivity in electrolyte"""
+        c_e = pybamm.maximum(c_e, 10)
```

Comment:
Out of curiosity, which kind of error do you get if you make this smaller?

Related to that, do we want to hard-code this or set some tolerances that the user can change?

Reply:
Making this smaller gives more casadi jacobian warnings. Yes, I will make it into a setting